cql-unit-tests/test/riskScore.test.ts

```diff
@@ -0,0 +1,246 @@
+import { buildDefaultClinicalAssessmentParameters, buildDefaultPatientDataParameters } from '../helpers/builders';
```

Comment:
It might be good to also do a test for "Total Risk Score" since I don't think we're testing it anywhere (unless I missed it!).

Reply:
We'll resolve this once it's in main